integration/templates/chart_operator_values.go

```diff
@@ -3,4 +3,9 @@
 package templates
 
 // ChartOperatorValues values required by chart-operator-chart.
-const ChartOperatorValues = `clusterDNSIP: 10.96.0.10`
+const ChartOperatorValues = `
+clusterDNSIP: 10.96.0.10
+resource:
+  tiller:
+    namespace: "giantswarm"
```

Comment:
Adding it otherwise it would use kube-system as a default.

Reply:
Makes sense. We need to always configure the namespace for tenant clusters. One day we will be tiller-less. Way better than server-less IMO 😄